gatsby-config.js

```diff
@@ -44,6 +44,10 @@ module.exports = {
           {
             title: 'Recipes',
             path: '/recipes/'
+          },
+          {
+            title: 'What\'s New',
+            path: '/new/'
```

Comment:
Please change the folder name to 'changelog'.

Reply:
Done